docs/en/chdb/install/python.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Installing chDB for Python
+sidebar_label: Python
+slug: /en/chdb/install/python
+description: How to install chDB for Python
+keywords: [chdb, embedded, clickhouse-lite, python, install]
+---
+
+# Installing chDB for Python
+
+## Requirements
+
+Python 3.8+ on macOS and Linux (x86_64 and ARM64)
+
+## Install
+
+```bash
+pip install chdb
+```
+
+## Usage
+
+CLI example:
+
+```python
+python3 -m chdb [SQL] [OutputFormat]
+```
+
+```python
+python3 -m chdb "SELECT 1, 'abc'" Pretty
+```
+
+Python file example:
+
+```python
+import chdb
+
+res = chdb.query("SELECT 1, 'abc'", "CSV")
+print(res, end="")
```

Comment:
We need introduce more about Session, DataFrame and UDF.
Some doc you can refer:
1. https://github.com/chdb-io/chdb/blob/main/README.md#data-input
1. https://github.com/chdb-io/chdb/blob/f14db2c6b3b8857ba009a20f2c5bb70aa3259a4a/chdb/udf/udf.py#L53
1. https://github.com/chdb-io/chdb/blob/f14db2c6b3b8857ba009a20f2c5bb70aa3259a4a/chdb/session/state.py#L9